cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -722,6 +852,21 @@ impl ABIBody for AArch64ABIBody {
         }
     }
 
+    fn gen_retval_area_setup(&self) -> Vec<Inst> {
```

Comment:
To avoid the Vec of at most one instruction (and the associated memory allocation), could this return either an `Option<Inst>` or a `SmallVec<[Inst, N]>`?